Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -37,6 +38,29 @@ extension BrowserViewController {
         popover.addsConvenientDismissalMargins = false
         popover.present(from: topToolbar.locationView.rewardsButton, on: self)
     }
+    
+    func configureRewardsLogs() {
+        if rewards == nil { return }
+        RewardsLogger.configure(logCallback: logCallback, withFlush: nil)
+    }
+    
+    private func logCallback(logLevel: LogLevel, line: Int32, file: String, data: String) {
+        if data.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty { return }
+        
+        let logger = Logger.rewardsLogger
+        // Rewards logger gives us full file path, extracting filename from it.
+        let fileName = (file as NSString).lastPathComponent
+        
+        let logOutput = "[\(fileName).\(line)] \(data)"
```

Comment:
Is showing filename and line something we can control with `Logger` somehow? So that we dont always show them regardless of log level?

Reply:
XCLogger `file` and `line` will point to this exact method, not to the source code of BraveRewards

Reply:
Ah, I see, lame… I wonder if we should add some sort of pref around whether or not to pre-pend file info? I guess if its just filename + line it won't be that bad… but still

Reply:
Yeah I was thinking about that, I can add some parameters with default values to the `configureRewardsLogs()`

Or I could see if XCLogger has fileinfo enabled and then redirect it to use the one provided by Rewards logger